solana/package.json

```diff
@@ -7,6 +7,8 @@
     },
     "dependencies": {
         "@certusone/wormhole-sdk": "^0.10.10",
+        "@wormhole-foundation/sdk-base": "^0.5.0-beta.9",
```

Comment:
Can you ensure that these match `package-lock.json`? There seems to be a mismatch in CI and I'm seeing the same thing when running the equivalent commands locally.

Reply:
yeah, realized that I updated both package.jsons but only ran npm install in the ci_test directory...